src/components/ManageUsers/manage-users.css

```diff
@@ -0,0 +1,84 @@
+.usersearch__h1 {
+  font-size: 16px;
+  color: #e3893e;
+  display: right;
+  font-weight: 200px;
+  margin-right: 10px;
+  text-align: right;
+}
+
+.usersearch__container {
+  padding: 16px;
+  min-height: 10vh;
```

Comment:
don't use `min-height` in that case. The height will be set automatically

Reply:
Removed